bluepysnap/frame_report.py

```diff
@@ -184,7 +238,7 @@ def simulation(self):
     @cached_property
     def population_names(self):
         """Returns the population names included in this report."""
-        return self._frame_reader.get_populations_names()
+        return sorted(self._frame_reader.get_populations_names())
```

Comment:
Should be reflected in a unit test (I'll check).

Reply:
this is implicitly done in the :
```
    def test_iter(self):
        assert sorted(list(self.test_obj)) == ["default", "default2"]
        for report in self.test_obj:
            isinstance(report, test_module.PopulationCompartmentReport)
```